srcpkgs/epson-inkjet-printer-escpr2/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'epson-inkjet-printer-escpr2'
+pkgname=epson-inkjet-printer-escpr2
+version=1.1.15
+revision=1
+archs="x86_64 i686 aarch64 armv7l"
+create_wrksrc=no
```

Comment:
Value of variable does not matter, but please change from misleading `no` to `yes`. Alternatively, remove create_wrksrc, build_wrksrc and skip top dir when unpacking tar.gz

Reply:
Removed that.

The next time I amend the commit , 
the create_wrksrc is gone and #build_wrksrc=${pkgname}-${version}

> skip top dir when unpacking tar.gz

Didn't understand that sorry

Reply:
I was wrong here, `create_wrksrc` and build_wrksrc are actually needed, otherwise masterdir is not cleaned up properly. Please define them back.

Reply:
Ok that's why I commented instead of deleting (when everything is okay I will delete the comments)